<?xml version="1.0"?>

<launch>


  <node name="inference_services" pkg="src2_object_detection" type="inference_service.py" output="screen">
  </node>

  <group ns="small_hauler_1">

    <param name="/use_sim_time" value="true"/>



    <node name="approach_base_station_service_hauler" pkg="src2_approach_services" type="approach_charging_station_service.py" output="screen">
      <param name="/robot_name" type="string" value="small_hauler_1"/>
    </node>

    <node name="object_estimation_service" pkg="src2_object_detection" type="object_estimation_service.py" output="screen">
    </node>

    <node name="to_point" pkg="src2_object_detection" type="object_to_point_cloud.py" respawn="true" output="screen">
    </node>

    <!-- <node name="plot" pkg="src2_object_detection" type="plot_boxes_real_time.py" output="screen">
    </node>

    <node name="approach_excavator" pkg="src2_object_detection" type="approach_excavator.py" output="screen">
    </node>

    <node name="approach_bin" pkg="src2_object_detection" type="approach_bin.py" output="screen">
        <param name="print_to_terminal" type="bool" value="True"/>
        <param name="rover_min_vel" type="double" value="0.6"/>
        <param name="approach_timeout" type="double" value="50"/>
        <param name="laser_range" type="double" value="3.0"/>
        <param name="rotational_speed" type="double" value="0.25"/>
    </node> -->

  </group>
</launch>
